### PR TITLE
[MINOR] improve(docs): use html entities

### PR DESCRIPTION
**Pull request by mchades**
### What changes were proposed in this pull request?

use html entities

### Why are the changes needed?

make the table clearer

### Does this PR introduce _any_ user-facing change?

no

### How was this patch tested?

local tested:

image

### CODE REVIEW

File: docs/manage-table-partition-using-gravitino.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ Although many catalogs inherently manage partitions automatically, there are sce
...
-| List Partitions       | YES                                                                | NO                                                                 | NO                 | NO                      |
-| Drop Partition        | [Coming Soon](https://github.com/datastrato/gravitino/issues/1655) | [Coming Soon](https://github.com/datastrato/gravitino/issues/1655) | NO                 | NO                      |
+| Operation             | Hive catalog                                                                  | Iceberg catalog                                                               | Jdbc-Mysql catalog | Jdbc-PostgreSQL catalog |
+|-----------------------|-------------------------------------------------------------------------------|-------------------------------------------------------------------------------|--------------------|-------------------------|
+| Add Partition         | &#10004;                                                                      | &#10008;                                                                      | &#10008;           | &#10008;                |
```
Comment (qqqttt123) [inline docs/manage-table-partition-using-gravitino.md:24]: It may be confusing if users only read from markdown document instead of website.

Comment (mchades) [reply docs/manage-table-partition-using-gravitino.md:24]: Markdown also can render the HTML entity:

image

Comment (qqqttt123) [reply docs/manage-table-partition-using-gravitino.md:24]: Cool!

**Review by qqqttt123 — APPROVED** [review]

File: docs/manage-metadata-using-gravitino.md [hunk 2]
```diff
@@ -831,22 +831,22 @@ The following is a table of the column default value that Gravitino supports for
...
-|---------------------|-----------------------------------------------------------------------------|
-| `hive`              | NO                                                                          |
-| `lakehouse-iceberg` | NO                                                                          |
-| `jdbc-mysql`        | YES with [limitations](./jdbc-mysql-catalog.md#table-column-auto-increment) |
-| `jdbc-postgresql`   | YES                                                                         |
+| Catalog provider    | Supported auto-increment                                                       |
+|---------------------|--------------------------------------------------------------------------------|
+| `hive`              | &#10008;                                                                       |
+| `lakehouse-iceberg` | &#10008;                                                                       |
+| `jdbc-mysql`        | &#10004;  ([limitations](./jdbc-mysql-catalog.md#table-column-auto-increment)) |
```
Comment (jerryshao) [inline docs/manage-metadata-using-gravitino.md:848]: There're two white spaces after "&#10004;"? If so, please remove one ws.

File: docs/manage-table-partition-using-gravitino.md [hunk 3]
```diff
@@ -19,13 +19,13 @@ Although many catalogs inherently manage partitions automatically, there are sce
...
+| Get Partition by Name | &#10004;                                                                      | &#10008;                                                                      | &#10008;           | &#10008;                |
+| List Partition Names  | &#10004;                                                                      | &#10008;                                                                      | &#10008;           | &#10008;                |
+| List Partitions       | &#10004;                                                                      | &#10008;                                                                      | &#10008;           | &#10008;                |
+| Drop Partition        | &#128640;([Coming Soon](https://github.com/datastrato/gravitino/issues/1655)) | &#128640;([Coming Soon](https://github.com/datastrato/gravitino/issues/1655)) | &#10008;           | &#10008;                |
```
Comment (jerryshao) [inline docs/manage-table-partition-using-gravitino.md:28]: Should you add a white space after "&#128640;"?

**Review by jerryshao — APPROVED** [review]
LGTM.

**Pull request merged**